### DISCUSSION TITLE
UV index Alerts

### DISCUSSION THREAD

**Discussion by diegol88** [breezy-weather/breezy-weather#1087] [Ideas]
### Describe your idea

Can we implement a notification when the UV index reaches a specific number? E.g.: Trigger an alert when the UV index is above high.

This could also be part of the morning report, like the expectation of the UV index during the day.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
The UV index varies during the day. The danger of UV depends on the hour and the index associated. There are plans to display risks of sunburn as part of #155 

As for notifications, there may be alerts provided by your national weather entity.

You can also use a widget to show max UV index of the day, which seems to be what you are interested in.
